src/main/java/com/aws/greengrass/builtin/services/lifecycle/LifecycleIPCEventStreamAgent.java

```diff
@@ -236,9 +233,10 @@ public DeferComponentUpdateResponse handleRequest(DeferComponentUpdateRequest re
                 if (deferComponentUpdateRequestFuture == null) {
                     throw new ServiceError("Time limit to respond to PreComponentUpdateEvent exceeded");
                 } else {
+                    log.atDebug().log("Processing deployment deferral from {} for deployment {}", serviceName,
```

Comment:
question: do we log somewhere that the deployment is deferred?

Reply:
Yes, https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/1171b2eb3e39917c1e9273d6d05d2f9eb344f8ba/src/main/java/com/aws/greengrass/lifecyclemanager/UpdateSystemPolicyService.java#L202